src/components/PDFView/index.js

```diff
@@ -97,6 +98,20 @@ class PDFView extends Component {
         });
     }
 
+    /**
+     * Sets attributes to list container.
+     * It unblocks a default scroll by keyboard of Chromium-based browsers.
+     * @param {Object|undefined} ref
+     */
+    setListAttributes(ref) {
+        if (!ref) {
+            return;
+        }
+
+        // eslint-disable-next-line no-param-reassign
+        ref.tabIndex = -1;
```

Comment:
Please add comment what's this for

Reply:
Done - df5c307.
Also, this method already includes JSCode with a description.